rbi/sorbet/tprops.rbi

```diff
@@ -146,8 +146,10 @@ module T::Props::PrettyPrintable::DecoratorMethods
 end
 
 module T::Props::Serializable
+  sig {params(hash: T::Hash[T.untyped, T.untyped], strict: T.nilable(T::Boolean)).returns(T.self_type)}
```

Comment:
This should probably be `T::Hash[String, T.untyped]`. That might need more fixes to existing code but it seems more likely to be helpful too - the most common kind of error I've seen here is passing Symbol-keyed hashes.